elastic/kibana
# [ResponseOps][Alerting] Ability run a rule on-demand

Resolves https://github.com/elastic/kibana/issues/50215

## Summary

- Added a button on the Rule and Connectors UI ("Rules" tab) that allows running a rule add hoc
<img width="289" alt="Screen Shot 2022-09-01 at 7 55 57 AM" src="https://user-images.githubusercontent.com/109488926/187946000-3627153c-44ed-45be-bb9d-1b3cb5eee9af.png">

- Leveraged the task manager's `runSoon` API

### Checklist


### To verify

- Go to the rules list page, and select to run a rule
- Verify that you see a toast saying that the rule is scheduled to run soon.
- You can refresh and verify that the rule is scheduled to run now
